modules/core/04-channel/v2/keeper/packet.go

```diff
@@ -68,12 +67,10 @@ func (k *Keeper) sendPacket(
 	if err != nil {
 		return 0, "", err
 	}
-	// check if packet is timed out on the receiving chain
-	// convert packet timeout to nanoseconds for now to use existing helper function
-	// TODO: Remove this workaround with Issue #7414: https://github.com/cosmos/ibc-go/issues/7414
-	timeout := channeltypes.NewTimeoutWithTimestamp(uint64(time.Unix(int64(packet.GetTimeoutTimestamp()), 0).UnixNano()))
-	if timeout.TimestampElapsed(latestTimestamp) {
-		return 0, "", errorsmod.Wrap(timeout.ErrTimeoutElapsed(latestHeight, latestTimestamp), "invalid packet timeout")
+
+	timeout := types.TimeoutTimestampToNanos(packet.TimeoutTimestamp)
+	if timeout < latestTimestamp {
```

Comment:
isn't that greater than equals? which is equivalent to less than, depending on what sides the arguments are on.

Reply:
Well, I'm stupid :)

Reply:
Would be fine with using <= here too, since if the timeout is equal to the latestTimestamp of latest counterparty header then, in practise by the time it reaches the counterparty the timeout would have elapsed